ssz/sedes/serializable.py

```diff
@@ -179,6 +179,17 @@ def __deepcopy__(self, *args):
     def root(self):
         return ssz.hash_tree_root(self)
 
+    def to_formatted_dict(self, int_as="int"):
+        from ssz.tools.io import FormattedDictIO
```

Comment:
Should be at the top of the file

Reply:
The intention is to import `tools` when needed since they are not core functions. But moving them to the top looks simpler.

Reply:
Ah, importing at the top has circular import issue.

Reply:
I don't think this should be included at all in the `Serializable` API at all.  `library.toolz` is meant to be completely separate from the main library functionality and we should have no problem implementing this functionality in a way that keeps `Serializable` from needing to be aware of it.


```python
# ssz/toolz/dump.py

def to_formatted_dict(value, sedes=None):
    if sedes is None:
        if isinstance(value, Serializable):
            sedes = type(value)
        else:
            raise Error("must provide sedes for non-Serializable"0
    ...  # do the conversion
```